pandas/sparse/series.py

```diff
@@ -570,27 +570,13 @@ def copy(self, deep=True):
         return self._constructor(new_data, sparse_index=self.sp_index,
                                  fill_value=self.fill_value).__finalize__(self)
 
+    @Appender(generic._shared_docs['reindex'] % _shared_doc_kwargs)
     def reindex(self, index=None, method=None, copy=True, limit=None,
                 **kwargs):
-        """
-        Conform SparseSeries to new Index
-
-        See Series.reindex docstring for general behavior
 
-        Returns
-        -------
-        reindexed : SparseSeries
-        """
```

Comment:
We are now using the same code as ``Series``, why shouldn't they work?! Anyway: I have now added tests for ``fill_value`` and ``tolerance``. It seems to me that ``level`` is untested even on normal ``Series``, I can add tests for both in a separate PR.